packages/rollup-plugin-html/test/rollup-plugin-html.test.ts

```diff
@@ -846,4 +846,42 @@ describe('rollup-plugin-html', () => {
       '<html><head></head><body><img src="./image-c.png"><link rel="stylesheet" href="./styles.css"><img src="./image-b.svg"></body></html>',
     );
   });
+
+  it.only('can inject a service worker registration script if injectServiceWorker and swDest are provided', async () => {
```

Comment:
```suggestion
  it('can inject a service worker registration script if injectServiceWorker and swDest are provided', async () => {
```

Reply:
still testing windows 🙈

Reply:
no problem just marking so we don't forget :D

Reply:
removed and all green